src/lib.rs

```diff
@@ -495,9 +495,16 @@ impl GenerateTasksService for GenerateTasksServiceImpl {
                 {
                     continue;
                 }
-
+                let last_versions_expansion = self
+                    .evg_config_utils
+                    .lookup_build_variant_expansion(UNIQUE_GEN_SUFFIX_EXPANSION, build_variant);
```

Comment:
Why is it named `last_versions_expansion` if it queries `UNIQUE_GEN_SUFFIX_EXPANSION`?

Reply:
good catch